### ISSUE TITLE
Transaction degiro csv doens0't have the same inputs as sample

### ISSUE THREAD

**Issue by rubengr** [dickwolff/Export-To-Ghostfolio#115]
Hi

I'm testing you code, but get this error:

```
npm run start degiro

> export-to-ghostfolio@0.18.2 start
> tsx ./src/manual.ts degiro

[i] Starting Export to Ghostfolio v0.18.2
[i] Restored 0 ISIN-symbol pairs and 0 symbols from cache..
[i] Processing file using DeGiro converter
[i] There is a new version of the DEGIRO converter available in public beta and we're looking for feedback!
[i] You can enable the new converter by setting the environment variable DEGIRO_FORCE_V3=true
[i] Read CSV file. Start processing..

[e] An error ocurred while trying to retrieve an empty symbol (line 5)!

[e] An error ocurred while processing.
[e] TypeError: progress.log is not a function
```
CSV columns aren't correct. These are my columns:
Data,Hora,Data Valor,Produto,ISIN,DescriÃ§Ã£o,T.,MudanÃ§a,,Saldo,,ID da Ordem

**Comment by dickwolff** [comment]
The columns itself do not matter as much (unless you’ve used the wrong export file but the headers to look correct to me), as they are language specific but the columns are always in the same position. So based on that the tool can infer the correct value while parsing.

The error on your log states it tries to find an empty symbol (with record on line 5 of your input). Can you maybe post this line here? Maybe this is a record that needs to be added to the ignore list.

**Comment by dickwolff** [comment]
Thank you. What kind of record is this? Should this affect your Ghostfolio (e.g. is it a Fee record), or can it be ignored (e.g. withdraw)?

**Comment by rubengr** [comment]
In this case is related to line 6 "03-10-2024,06:50,02-10-2024,,,Levantamento de divisa,1.1072,USD,-5.77,USD,0.00,".
It was removed 5.77 USD but the account is in € so the line 5 is the convertion of 5,77USD to Euro.
I have other similar:
Line 11: 19-09-2024,20:20,19-09-2024,APPLE INC,US0378331005,Levantamento de divisa,1.1191,USD,-457.52,USD,0.00,711b1ad7-a370-4c88-bc86-074aada4a278
Line 12: 19-09-2024,20:20,19-09-2024,APPLE INC,US0378331005,CrÃ©dito de divisa,,EUR,408.83,EUR,638.81,711b1ad7-a370-4c88-bc86-074aada4a278
Line 13: 19-09-2024,20:20,19-09-2024,APPLE INC,US0378331005,ComissÃµes de transaÃ§Ã£o DEGIRO e/ou taxas de terceiros,,EUR,-2.00,EUR,229.98,711b1ad7-a370-4c88-bc86-074aada4a278
Line 14: 19-09-2024,20:20,19-09-2024,APPLE INC,US0378331005,"Venda 2 Apple Inc@228,76 USD (US0378331005)",,USD,457.52,USD,457.52,711b1ad7-a370-4c88-bc86-074aada4a278

In this case is the selling of 2 shares of apple in line 14. Line 13 is the comission. Line 12 is the convertion of USD to € added to account and line 11 is the same value in USD.

**Comment by dickwolff** [comment]
Thanks for the information! If I interpret these correctly these are similar to records in my Dutch export where there is this thing with currency conversions after USD > EUR transactions and I’ve ignored those. So I propose to add these records to the ignore list as well.

**Comment by dickwolff** [comment]
Should be fixed in 0.18.4.
